### ISSUE TITLE
Bulk invoice printing

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/openfoodnetwork#1675]
It would be really useful for Hubs to be able to select all their orders for the week with some kind of "print all" option, rather than navigating through each order and then each invoice individually.

**Comment by myriamboure** [comment]
Yes, and actually ideally, we should be able to mark an order as "delivered" (do that in bulk for an OC for ex) and then automatically the invoice should be generated and could be accessible from the user account ;-) And the  hub manager should be able to print them in bulk for an OC also, yes.

**Comment by kirstenalarsen** [comment]
work did start on a full solution to this, but obviously got put to the side. There was (is) a design for angularised 'Orders index' page that enabled management of multiple orders concurrently, including selecting and multiple print / send of invoices. Would be good to see this happem, but maybe best wait till after spree upgrade?

**Comment by sauloperez** [comment]
> but maybe best wait till after spree upgrade?

Definitely. We don't want to waste our efforts implementing features that might need to change in newer versions of Spree.

**Comment by sstead** [comment]
Hey @sauloperez just checking in, is this still blocked by spree? Do you know any more details? Thanks!

**Comment by sigmundpetersen** [comment]
Hey @daniellemoorhead didn't we agree on removing those country labels? 😇 
https://community.openfoodnetwork.org/t/how-we-manage-things-in-github-labels/1097

**Comment by myriamboure** [comment]
Sorry to be picky but this is not a bug, this is a new feature to be built, so this issue should be closed for now until we decide to prioritize (maybe tomorrow :-)) and then incepted properly together before we open all the stories needed. So I added that to https://community.openfoodnetwork.org/t/make-invoicing-operations-smooth-and-easy-for-hub-managers-and-customers/1200 and I'm closing this for now, but please don't worry if we decide to prioritize tomorrow we will open a epic and incept (ask question about Spree, study different UX options, etc) I know @kirstenalarsen already put some votes on it ;-) @daniellemoorhead I'm surprised you did kick asses here, process master ;-)

**Comment by daniellemoorhead** [comment]
Absolutely agree with the approach. I didn't see the other item until I
went into the freature backlog, and then ran out of time to look at this
one and close it. So this is good.
